readme.md

```diff
@@ -1,11 +1,88 @@
-# [WIP] Spade Tenant Service
+<div align=center>
+<img src="docs/logo.png" width=300 />
 
+# Spade Tenant Services
+</div>
 
-## Generating Schema
-`DB_ALLOW_MIRGATIONS=true go run . > out.sql`
 
+Tenant APIs and services for the [Spade](https://github.com/data-preservation-programs/spade) - Filecoin network storage proposal management service
 
-## Generating Swagger
+The Tenant services handle all aspects of Spade Storage Proposals from the Tenant side. At a high level, this includes interfaces to manage the following:
+- Policies
+- Storage Contracts
+- Collections
+- Content (i.e, sources of Filecoin Pieces)
+- Replication Constraints
+- Storage Provider SLA Compliance
+
+A high level architecture diagram of the Tenant functionalities and their relationships can be found [here](https://miro.com/app/board/uXjVMwfo8TM=/?share_link_id=267050417898) 
```

Comment:
This is marked as internal in miro. Should it be shared?

Reply:
It's publically viewable with the link, should be okay.